controller/api/permissions.py

```diff
@@ -97,7 +97,19 @@ class HasRegistrationAuth(permissions.BasePermission):
     Checks to see if registration is enabled
     """
     def has_permission(self, request, view):
-        return settings.REGISTRATION_ENABLED
+        """
+        If settings.REGISTRATION_MODE does not exist, such as during a test, return True
+        Return `True` if permission is granted, `False` otherwise. Defaults to True
+        """
+        try:
+            if settings.REGISTRATION_MODE == 'disabled':
+                return False
+            elif settings.REGISTRATION_MODE == 'adminOnly':
```

Comment:
I'd prefer `admin_only` as the type here. It's more pythonic to use underscores rather than camelcase.